infra/shared/storekeys.bicep

```diff
@@ -3,8 +3,8 @@ param azureOpenAIName string = ''
 param azureAISearchName string = ''
 param rgName string = ''
 // Do not use _ in the key names as it is not allowed in the key vault secret name
-param openAIKeyName string = 'AZURE-OPENAI-KEY'
-param searchKeyName string = 'AZURE-SEARCH-KEY'
+param openAIKeyName string = 'openai-api-key'
```

Comment:
So the reason this is renamed here is to make it compatible with whatever is defined in .env already. This way, whether the value is created in .env or in secret, the code does not change. If I now rename this, I would have to let all users know that they need to change their .env files.

Otherwise, I agree with you!